packages/lib/src/index.spec.ts

```diff
@@ -5,18 +5,20 @@ describe('packages/lib', () => {
     test('Basic usage', () => {
       type Test = OneOf<{ foo: number }, { bar: string }>;
 
-      const fooOkay: Test = { foo: 42 };
-      // @ts-expect-error
-      const fooBad: Test = { foo: 'hello' };
+      (function* (): Generator<Test, void, never> {
```

Comment:
What's going on here?

Reply:
typescript was complaining about unused variables and there's no way for me to silence it without also silencing other type errors. this is a small hack to assert the type of a bunch of values without having to give them them names since all the yielded values must confirm to the declared generator type